typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx

```diff
@@ -46,20 +46,22 @@ export const ItemList: React.FC<Prop> = (props) => {
   }, [reload]);
 
   return (
-    <div>
-      {items.map((item) => {
-        return (
+    <div className="grid-container">
+      {items && items.length > 0 && (
```

Comment:
[nits]
条件は変数に入れてあげると少し読みやすくなります〜！

```typescript
// return の前に定義
const existsItems = items && items.length > 0
```
```suggestion
    <div className="grid-container">
      {existsItems && (
```

Reply:
なるほど！勉強になりました、ありがとうございます🥰
こちらで修正しました。[dae55eb](https://github.com/dongurikoko/mercari-build-training/pull/9/commits/dae55eb8c2797ec0a170234f3c1d0a4f0e7ef356)